storage/file.go

```diff
@@ -92,40 +292,176 @@ func pathForFileShare(name string) string {
 // See https://msdn.microsoft.com/en-us/library/azure/dd179352.aspx
```

Comment:
This is the correct link `https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/list-shares`